app/controllers/homepage_controller.rb

```diff
@@ -6,5 +6,14 @@ def index
     @newbooking = Newbooking.all
     @totalbookings = @newbooking.size
     @totalmoney = @newbooking.reduce(0) { |sum, booking| sum + booking.service.price }
+    return unless params[:pet_id]
+
+    pet = Pet.find(params[:pet_id])
+    customer = pet.customer
+    make_quick_booking(pet, customer)
```

Comment:
something to think about - this method doesn't need to know the customer or the pet other than to tell them to `make_quick_booking`.

It might be better to encapsulate that logic to find the pet and customer in the method that needs it instead of pulling it out here.